app/javascript/components/layout/Header.jsx

```diff
@@ -58,10 +58,7 @@ const Header = () => {
                         <div className="menu">
                             <ul>
                                 <li>
-                                    <a href="/">Upcoming Meetup</a>
-                                </li>
-                                <li>
-                                    <a href="/meetups">Past Meetups</a>
+                                    <a href="/jobs">Job Board</a>
```

Comment:
Notes on these changes:
* For meetups, the "Past Meetups" page is technically done, but the individual meetups page is not (currently in progress, see #7). I'd rather hide these pages for now and then add them back to the header once they're fully functional!
* Job board is done but hasn't been in the header yet 🎉

Reply:
Can we add putting the header back as part of the acceptance criteria of #7 ?